wisembly/amqp-bundle/1.3/post-install.txt

```diff
@@ -0,0 +1,8 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * <fg=blue>Add</> "gates" to the bundle configuration
```

Comment:
It's actually the concept of the bundle, not AMQP. 

I thought I documented that, but it doesn't seem to be the case, so indeed, only those familiar with the bundle (not a lot of people currently TBH) would understand that. I'll document that.